contents/docs/data/user-properties.mdx

```diff
@@ -0,0 +1,177 @@
+---
+title: User properties
+sidebarTitle: User properties
+featuredImage: ./images/docs-properties.png
+nextPage: ./actions-and-insights.mdx
+sidebar: Docs
+showTitle: true
+---
+
+import UserPropertiesHowToSet from "./\_snippets/user-properties-how-to-set.mdx"
+import UserPropertiesSetVsSetOnce from "./\_snippets/user-properties-set-vs-set-once.mdx"
+
+User properties are a powerful feature in PostHog which enable you to better manage, analyze, and control data. You can create [filters](/docs/product-analytics/trends#filtering-events-based-on-properties) or [cohorts](/docs/data/cohorts) based on user properties, which can then be used to [create insights](/docs/product-analytics/insights), [feature flags](/docs/feature-flags), and more. 
+
+## How to set user properties
+
+<UserPropertiesHowToSet/>
+
+### What is the difference between `set` and `set_once`?
+
+<UserPropertiesSetVsSetOnce/>
+
+### How to remove user properties
+
+Similarly to how you set properties, you can use `$unset` to remove properties by calling `capture` and including an array of properties keys:
+
+<MultiLanguage selector="tabs">
+
+```javascript
+posthog.capture(
+    'event_name', 
+    { 
+        $unset: ['email'],
+    }
+)
+```
+
+```node
+client.capture({
+    distinctId: 'distinct_id',
+    event: 'event_name',
+    properties: {
+        $unset: ['email'],
+    },
+})
+```
+
+```python
+posthog.capture(
+    'distinct_id',
+    event='event_name',
+    properties={
+        '$unset': ['email'],
+        '$set_once': {'initial_url': '/blog'}
+    }
+)
+```
+
+```php
+PostHog::capture(array(
+    'distinctId' => 'distinct_id',
+    'event' => 'event_name',
+    'properties' => array(
+        '$unset' => array(
+           'email'
+        )
+    )
+));
+```
+
+```ruby
+posthog.capture({
+    distinct_id: 'distinct_id',
+    event: 'event_name',
+    properties: {
+        '$unset': ['email']
+    }
+})
+```
+
+```go
+client.Enqueue(posthog.Capture{
+    DistinctId: "distinct_id",
+    Event:      "event_name",
+    Properties: map[string]interface{}{
+        "$unset": []string{"email"},
+    },
+})
+```
+
+```java
+posthog.capture("distinct_id", "event_name", new HashMap<String, Object>() {
+  {
+    put("$unset", new String[]{"name"});
+  }
+});
+```
+
+</MultiLanguage>
+
+
+## How to set group properties
+
+In the same way that every user can have properties associated with them, every [group](/docs/product-analytics/group-analytics) can have properties associated with it. You can find details on [how to set group properties](/docs/product-analytics/group-analytics#setting-and-updating-group-properties) in the groups docs.
```

Comment:
I gave this some thought, and I dont think we need to tell people about this implementation detail. I would prefer to be concise, to avoid potential confsuion